camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/common/EtlCounts.java

```diff
@@ -275,6 +276,8 @@ private void produceCount(String brokerList, ArrayList<byte[]> monitorSet) {
           }
         }
       }
+    } catch (Exception e) {
+      throw new RuntimeException("failed to publish counts to kafka: " + e.getMessage());
```

Comment:
You probably want to change this to:

throw new RuntimeException("failed to publish counts to Kafka: " + e.getMessage(), e);

So you get the full stack trace of the original exception